src/org/jitsi/impl/neomedia/MediaStreamImpl.java

```diff
@@ -1067,6 +1061,12 @@ private TransformEngineChain createTransformEngineChain()
             engineChain.add(mediaStreamTrackReceiver);
         }
 
+        // Debug
+        debugTransformEngine
```

Comment:
so this mean the debug engine runs (close to) first on the ingress side, and (close to) last on the egress, right?